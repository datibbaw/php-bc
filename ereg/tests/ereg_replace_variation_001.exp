*** Testing ereg_replace() : usage variations ***
Error: 8 - Undefined variable: undefined_var, %s(64)
Error: 8 - Undefined variable: unset_var, %s(67)

Arg value 0 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)

Arg value 1 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"

Arg value 12345 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"

Arg value -2345 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"

Arg value 10.5 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"

Arg value -10.5 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"

Arg value 101234567000 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"

Arg value 1.07654321E-9 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)

Arg value 0.5 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)
Error: 8 - Array to string conversion, (%d)

Arg value Array 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)
Error: 8 - Array to string conversion, (%d)

Arg value Array 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"
Error: 8 - Array to string conversion, (%d)

Arg value Array 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"
Error: 8 - Array to string conversion, (%d)

Arg value Array 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"
Error: 8 - Array to string conversion, (%d)

Arg value Array 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)

Arg value 1 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)

Arg value 1 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(8) "original"

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)
Error: 4096 - Object of class stdClass could not be converted to string, %s(73)

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 8 - Object of class stdClass could not be converted to int, %s(74)
string(8) "original"

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace(): REG_EMPTY, %s(74)
bool(false)
Done