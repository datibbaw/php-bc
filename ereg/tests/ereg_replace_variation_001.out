*** Testing ereg_replace() : usage variations ***
Error: 8 - Undefined variable: undefined_var, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_001.php(64)
Error: 8 - Undefined variable: unset_var, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_001.php(67)

Arg value 0 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"

Arg value 1 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"

Arg value 12345 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"

Arg value -2345 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"

Arg value 10.5 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"

Arg value -10.5 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"

Arg value 101234567000 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"

Arg value 1.07654321E-9 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"

Arg value 0.5 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_001.php(73)

Arg value Array 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
string(35) "newonewrnewinewgnewinewnnewanewlnew"
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_001.php(73)

Arg value Array 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
string(35) "newonewrnewinewgnewinewnnewanewlnew"
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_001.php(73)

Arg value Array 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
string(35) "newonewrnewinewgnewinewnnewanewlnew"
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_001.php(73)

Arg value Array 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
string(35) "newonewrnewinewgnewinewnnewanewlnew"
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_001.php(73)

Arg value Array 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
string(35) "newonewrnewinewgnewinewnnewanewlnew"

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(35) "newonewrnewinewgnewinewnnewanewlnew"

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(35) "newonewrnewinewgnewinewnnewanewlnew"

Arg value 1 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(35) "newonewrnewinewgnewinewnnewanewlnew"

Arg value 1 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(8) "original"

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(35) "newonewrnewinewgnewinewnnewanewlnew"

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(35) "newonewrnewinewgnewinewnnewanewlnew"

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(35) "newonewrnewinewgnewinewnnewanewlnew"
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_001.php(73)

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/ereg.php(46)
string(35) "newonewrnewinewgnewinewnnewanewlnew"

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(35) "newonewrnewinewgnewinewnnewanewlnew"

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(35) "newonewrnewinewgnewinewnnewanewlnew"
Done